modules/pubmaticBidAdapter.js

```diff
@@ -217,6 +276,221 @@ function _checkParamDataType(key, value, datatype) {
   }
 }
 
+function _createNativeRequest(params) {
+  var nativeRequestObject = {
+    assets: []
+  };
+  for (var key in params) {
+    if (params.hasOwnProperty(key)) {
```

Comment:
is this check required? since we are reading key from params in the first place?